orderValidation.js

```diff
@@ -114,7 +104,7 @@ const validateExpiryTime = (expiry) => {
   )
 }
 
-const validateMarginAfterTrade = ({ currentPosition, trade, feeRate, maxLeverage } = {}) => {
+const validateMarginAfterTrade = ({ currentPosition, trade, feeRate, maxLeverage, fairPrice } = {}) => {
   const positionAfterTrade = calcPositionAfterTrade({
```

Comment:
assuming you are using tradePrice here, its good to go

Reply:
yeah position after trade is calculated with actual trade price, all margin calculations use fair price instead